web/static/js/components/idea_read_only_content.jsx

```diff
@@ -18,6 +19,8 @@ const IdeaReadOnlyContent = props => {
         stage={stage}
       />
       <span data-hj-masked>{ idea.body }</span>
+      { hasAssignee && <span className={styles.assignee}> ({assignee.given_name}</span> }
```

Comment:
no need for multiple spans here, as the user has a `name` attribute (provided for convenience by google in their oauth info) that is a concatenation of the user's first and last name.

Reply:
👀 !

Reply:
done :)